perf/benchmark/configs/istio/ext-authz/README.md

```diff
@@ -0,0 +1,21 @@
+# Ext-authz Benchmarking
+This directory contains scripts and configurations for ext-authz benchmarking.
+## Prerequisites
+1. See [Istio Performance Benchmarking](https://github.com/istio/tools/tree/master/perf/benchmark) for environment setup.
+2. Ext-authz setup: 
```

Comment:
can you add ext authz setup in your run.sh, let's make it fully automated.

Reply:
ext-authz setup involves editing a config map, and the environment setup has made the config map too complicated to edit it in script